### PR TITLE
Create-scorecards-for-all-of-the-app-types-and-verify-that-they-work-issue-58

### PR DESCRIPTION
**Pull request by randytpierce**
branch Create-scorecards-for-all-of-the-app-types-and-verify-that-they-work-issue-58

Add update for status and processedAt, improved README and added some comments

### CODE REVIEW

File: pkg/manager/manager_test.go [hunk 1]
```diff
@@ -50,10 +50,10 @@ func getTestDoc(mngr *Manager) (map[string]interface{}, error) {
 	return scorecardCB, nil
 }
 
-func upsertTestDoc(mngr *Manager) error {
+func upsertTestDoc(mngr *Manager, test_doc_file string, test_doc_id string) error {
```
Comment (ian-noaa) [inline pkg/manager/manager_test.go:53]: That'll be nice to have!

File: pkg/manager/manager_test.go [hunk 2]
```diff
@@ -325,7 +325,45 @@ func Test_runManager(t *testing.T) {
...
+	}
+	err = manager.Run()
+	if err != nil {
+		t.Fatal(fmt.Sprint("manager test run error ", err))
+	}
+	elapsed := time.Since(start)
+	fmt.Printf("The test took combined %s", elapsed)
+}
+
+func Test_flipped_runManager(t *testing.T) {
```
Comment (ian-noaa) [inline pkg/manager/manager_test.go:345]: Just for my knowledge - what's a "flipped" scorecard?

File: pkg/manager/manager.go [hunk 3]
```diff
@@ -435,10 +436,43 @@ func (mngr Manager) Run() (err error) {
...
+	if err != nil {
+		return fmt.Errorf("error setting processedAt %q", err)
+	}
+	elapsed := time.Since(start)
+	log.Printf("This run processed: %v cells in %v", cellCount, elapsed)
+	return nil
+}
+
+func (mngr Manager) SetStatus(status string) (err error) {
+	stmnt := "UPDATE vxdata._default.SCORECARD SET status = \"ready\" where meta().id=\"" + mngr.documentID + "\";"
```
[the remark this answers is not in the document]
Comment (randytpierce) [reply pkg/manager/manager.go:461]: Good catch

File: README.md [hunk 4]
```diff
@@ -34,6 +34,49 @@ brew install golangci-lint # If not installed already
...
+To build the cmdline cli for mac use the following command...
+
+```@bash
+GOOS=darwin GOARCH=amd64 go build -o bin/mac-process cmd/cli/process.go
+```
+
+To build the cli for linux use ...
+
+```@bash
+GOOS=darwin GOARCH=amd64 go build -o bin/mac-process cmd/cli/process.go
```
[the remark this answers is not in the document]
Comment (randytpierce) [reply README.md:47]: another good catch

[the remark this answers is not in the document]
Comment (randytpierce) [reply README.md:47]: ok

File: pkg/manager/manager_test.go (hunk 2 above)
Comment (randytpierce) [reply pkg/manager/manager_test.go:345]: The control data source and the experimental data source are reversed. It should cause the difference direction to change, I think.

**Comment by ian-noaa** [comment]
Closing in favor of #71

**Pull request closed without merging**